spec/Section 3 -- Type System.md

```diff
@@ -1692,6 +1686,20 @@ While defining a directive, it must not reference itself directly or indirectly:
 directive @invalidExample(arg: String @invalidExample) on ARGUMENT_DEFINITION
 ```
 
+**Built-in Directives**
+
+GraphQL implementations should provide the `@skip` and `@include` directives.
+When returning the set of directives from the `__Schema` introspection type, both
+`@skip` and `@include` directives must be included.
+
+GraphQL implementations that support the type system definition language must
+provide the `@deprecated` directive if representing deprecated portions of
+the schema, in this case `@deprecated` directive may be included when returning
+the set of directives from the `__Schema` introspection type.
+
+When representing a GraphQL schema using the type system definition language,
+both `@skip`, `@include` and `@deprecated` directives can be omitted for brevity.
```

Comment:
Since now we have definition for `built-in directives` it's better to use it:
```suggestion
 built-in directives can be omitted for brevity.
```

Also, I checked #597 and says:
> all built-in scalars must be omitted for brevity.

So I think you were right initially and we should use `must` for consistency.
Sorry for the confusion 🤦‍♂ 

```suggestion
built-in directives must be omitted for brevity.
```

Reply:
`should` is used in the scalars section, so here I used it too